_posts/memory-creation-overflow-bug.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Solidity Memory Array Creation Overflow Bug"
+date:   2020-04-02
+author: Solidity Team
+---
+
+# Solidity Memory Array Creation Overflow Bug Announcement
```

Comment:
This title is not needed, hence it results in a double title (see https://solidity.ethereum.org/2019/06/25/solidity-storage-array-bugs/); I will adjust it in the old posts. Please remove the title here.